modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -1,12 +1,20 @@
 <?php
 
 /**
- * Implements hook_page_attachments.
+ * @file
+ * stanford_profile_styles.module
  */
-function stanford_profile_styles_page_attachments(array &$page) {
+
+/**
+ * Implements hook_page_attachments().
+ */
+/**
+ * Implements hook_page_attachments_alter().
+ */
+function stanford_profile_styles_page_attachments_alter(array &$attachments) {
```

Comment:
Why did this change to hook_alter?

Reply:
It didn't actually work. Once I changed to an alter the css got added. Same for the Stanford page css

Reply:
It's working as hook_page_attachments for me. I see the stylesheets in the head. Can you try again?

Reply:
hmm yeah now it is working again. wth. :man_shrugging: 
I'll try to fix up the tests now. something in caravan is loading `8.8.0-alpha1` of core for some reason.